src/stock_str.rs

```diff
@@ -275,6 +275,42 @@ pub enum StockMessage {
 
     #[strum(props(fallback = "Download maximum available until %1$s"))]
     DownloadAvailability = 100,
+
+    #[strum(props(fallback = "Inbox"))]
+    Inbox = 103,
+
+    #[strum(props(fallback = "Outbox"))]
+    Outbox = 104,
+
+    #[strum(props(fallback = "Storage on %1$s"))]
+    StorageOnDomain = 105,
+
+    #[strum(props(fallback = "One moment…"))]
+    OneMoment = 106,
+
+    #[strum(props(fallback = "Connected"))]
+    Connected = 107,
+
+    #[strum(props(fallback = "Connecting…"))]
+    Connecting = 108,
+
+    #[strum(props(fallback = "Updating…"))]
+    Updating = 109,
+
+    #[strum(props(fallback = "Sending…"))]
+    Sending = 110,
+
+    #[strum(props(fallback = "Your last message was sent successfully."))]
+    LastMsgSentSuccessfully = 111,
+
+    #[strum(props(fallback = "Error: %1$s"))]
+    Error = 112,
```

Comment:
NB: This is the same as `DC_STR_CONFIGURATION_FAILED`. No need to change anything here, just something I noticed.

Reply:
the ui-translations contain a "Configuration failed", which was removed here for $reasons. also there seems to be a linebreak in the "Configuration failed" string.

so, i would use a new, clear string here. we can maybe change other strings to this one and abandon the "Configuration failed" at some point. but that is not really important, agree :)